ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsApi.kt

```diff
@@ -0,0 +1,18 @@
+package com.braintreepayments.api
+
+import com.braintreepayments.api.findeligiblepayments.FindEligiblePaymentsApiResult
+import com.braintreepayments.api.findeligiblepayments.FindEligiblePaymentsApi
+import com.braintreepayments.api.findeligiblepayments.FindEligiblePaymentsApiRequest
+
+/**
+ * [ShopperInsightsApi] is a wrapper class for all api related classes that
+ * [ShopperInsightsClient] depends on.
+ */
+internal class ShopperInsightsApi(
+    private val paymentsApi: FindEligiblePaymentsApi
```

Comment:
```suggestion
    private val findEligiblePaymentsApi: FindEligiblePaymentsApi
```

Would we want to match the variable name to the class name here?

Reply:
fixed!